Samples/PowerShell/functions/func_ModernDashboards.ps1

```diff
@@ -4,6 +4,134 @@
 #Requires -Module @{ ModuleName = 'SwisPowerShell'; ModuleVersion = '3.0.0' }
 #Requires -Version 7
 
+<#
+.Synopsis
+    List Modern Dashboards from SolarWinds Orion system
+.DESCRIPTION
+    Connects to SolarWinds Information Service and enumerates the user-defined Modern Dashboards
+.EXAMPLE
+    $SwisConnection = Connect-Swis -Hostname "192.168.11.165" -Username "admin" -Password "MyComplexPassword"
+    PS C:\Exports> Get-SwisModernDashboard -SwisConnection $SwisConnection
+
+    This command list all of the Modern Dashboards
+.EXAMPLE
+    $SwisConnection = Connect-Swis -Hostname "192.168.11.165" -Username "admin" -Password "MyComplexPassword"
+    PS C:\> Get-ModernDashboard -SwisConnection $SwisConnection -DashboardId 9 
```

Comment:
It's a bit confusing to have some examples refer to the function by its primary name and some by its alias.  I would have all of them use the primary name.  And since `Get-SwisModernDashboard` is new, does it even need an alias?

Reply:
Technically, it doesn't need an alias since it's net-new, but I figured it might be better to use similar syntax everywhere.  The alias isn't hurting anything, so I figured I should add it for symmetry with the other functions.